### PR TITLE
Faq branch

### PR DESCRIPTION
**Pull request by baokiddy**
Updated this branch with all documentation created in HackMD [27.12.22]

### CODE REVIEW

**Comment by sarob** [comment]
Check out my changes. The .md extension signals that the content is markdown. I will update the PR #38 to match the new files. Let me know if this makes sense to you.

File: docs/faq/glossary.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+POAP is commonly used to describe a collectible that can be used, for example, to certify the presence in a physical or online event, the participation in a hackathon. Broadly, it describes the gift of an event organizer to people attending the event.
+You can find more details about POAP in the [documentation](https://poap.zendesk.com/hc/en-us/articles/9494654007437-What-Is-POAP-)
+
+## Lego
+A Lego is a composable unit that performs a definite action. You can see it as a piece of software in which, for a given input, it will return an output. Gitcoin Passport scorer is an example of Sybil detection lego developed by Gitcoin that, from the stamps verified by a user returns a trust score.
+
+You can find more details in this Gitcoin blog post: [Anti-Sybil Legos](https://gov.gitcoin.co/t/anti-sybil-legos/12265)
+
+## What is Exploratory Data Analysis (EDA)
+(**V0.0 @G-r-ay : Review needed**)
```
Comment (poupou-web3) [inline docs/faq/glossary.md:44]: Should be removed

Comment (sarob) [reply docs/faq/glossary.md:44]: Is it redundant, wrong, or something else?

Comment (sarob) [reply docs/faq/glossary.md:44]: Take a look at #38 as these two PRs can be merged together now.

Comment (baokiddy) [reply docs/faq/glossary.md:44]: I think clarification is that the V0.0 is no longer needed in this version. Let me make an update and push to update this.

Also @sarob reviewed #38 and looking good. Thank you for the update!

**Review by baokiddy — COMMENTED** [review]
Updated the glossary document and checked to make sure update works with #38. Should be all good to merge after a final review.

**Comment by sarob** [comment]
looks good. Thanks!

**Review by sarob — APPROVED** [review]
Reviewed the files and ready for merge

**Pull request merged**